api/office_structure/schema.py

```diff
@@ -61,6 +61,20 @@ def mutate(self, info, id):
         return DeleteNode(node=node)
 
 
+class Query(graphene.ObjectType):
+    get_node_by_id = graphene.Field(
```

Comment:
Variable names should not be a verb in nature like `get_node_by_id` same to a declaration in line 72

Reply:
This has been handled